### ISSUE TITLE
An inline image in the table blocks the cross-cell navigation.

### ISSUE THREAD

**Issue by magda-chrzescian** [ckeditor/ckeditor5#9380]
## 📝 Provide detailed reproduction steps (if any)

1. Open the `i/2052-inline-images` branch.
2. Run and open the all-features manual test.
3. Set the cursor at the end of the text in a table cell and insert an image.
4. Select the image and try to navigate to the next cell on the right/bottom.
5. Set the cursor at the beginning of the text in a table cell and insert an image.
6. Select the image and try to navigate to the previous cell on the left/top.

### ✔️ Expected result

It should be possible to go to the previous/next cell with the keyboard.

### ❌ Actual result

The selection returns to the inline image instead of moving to the next/previous cell.

![2021-03-29 11 50 22](https://user-images.githubusercontent.com/14334820/112820066-89b01300-9085-11eb-825f-d8622d14547c.gif)

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by AnnaTomanek** [comment]
Does not look like an MVP thing, more of a follow-up after the initial release.
